monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {Paper, TableBody, TableCell, TableHead, TableRow, Typography} from "@material-ui/core";
+import {withStyles} from "@material-ui/core/styles";
+import Link from "@material-ui/core/Link";
+import React from "react";
+import Table from "@material-ui/core/Table";
+import {useHistory} from "react-router-dom";
+
+const styles = theme => ({
+    policyContent: {
+        paddingLeft: '25%',
+        paddingRight: '25%',
+        paddingTop: '3%',
+        paddingBottom: '3%',
+        fontsize: '14px'
+    },
+});
+
+function CookiePolicy(props) {
+
+    const {classes} = props;
+    const history = useHistory();
+    return (
+        <Paper className={classes.policyContent}>
+
+            <Typography variant="h3">WSO2 Micro Integrator Dashboard - Cookie Policy</Typography>
+            <Link variant="h4"
+                  href="https://apim.docs.wso2.com/en/latest/observe/mi-observe/working-with-monitoring-dashboard/"
+                  style={{color: "#ff5000"}}>WSO2 Micro Integrator Dashboard</Link>
+
+            <Typography>
+                WSO2 Micro Integrator Dashboard monitors running Micro Integrator instances (Single or Cluster
+                Mode) and facilitates
+                performing various management and administration tasks related to deployed artifacts.
+            </Typography>
+
+            <h2 id="cookie-policy">Cookie Policy</h2>
+            <Typography>
+                WSO2 Micro Integrator Dashboard uses cookies so that it can provide the best user experience for
+                you and identify you for security purposes. If you disable cookies, some of the services will
+                (most probably) be
```

Comment:
Let's avoid "most probably" here. I think this was discussed elsewhere.

Reply:
removed the entire section